packages/driver/cypress/e2e/commands/screenshot.cy.js

```diff
@@ -299,6 +299,9 @@ describe('src/cy/commands/screenshot', () => {
       cy.screenshot().then(() => {
         expect(Cypress.automation.withArgs('take:screenshot').args[0][1].name).to.be.undefined
       })
+
+      // Reset the runnable title to the test name so we can close the span for the test.
+      runnable.title = 'sets name to undefined when not passed name'
```

Comment:
runnable.id isn't much better its' just an int that counts up for each spec, the title will help more. even if we loose the history of the test when it's title changes... it just doesn't have a uuid that we can track.

Reply:
What if two tests have the same title path?

Reply:
wonder if we can use a title + a timestamp?

Reply:
> What if two tests have the same title path?

then you don't get good telemetry data on those tests... i'd recommend you don't do this. We could add the runnable.id as an attribute to kinda tell the tests apart

> wonder if we can use a title + a timestamp?

This would make it so we have a hard time comparing tests from run to run.